<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayArena</class>
 <widget class="QMainWindow" name="PlayArena">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1926</width>
    <height>1086</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlayArena</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="handWidget0" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>760</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="deckImageWidget" native="true">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>100</y>
      <width>120</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="handWidget1" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="handWidget2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="handWidget3" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="BetHistory">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>170</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <column>
     <property name="text">
      <string>Comb</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Suit</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="handWidget4" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>270</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="handWidget5" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>510</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="endwidget" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QLabel" name="endMesssadge">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>20</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>321</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="cardPull"/>
    </widget>
    <widget class="QPushButton" name="startRound">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>88</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:#854DFA}</string>
     </property>
     <property name="text">
      <string>DEAL</string>
     </property>
    </widget>
    <widget class="QLabel" name="messadge">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>40</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>help</string>
    </property>
   </widget>
   <widget class="QLabel" name="activeIndecator_0">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>826</y>
      <width>66</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="activeIndecator_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>66</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="avtiveIndecator_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>66</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="avtiveIndecator_4">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>200</y>
      <width>66</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="avtiveIndecator_5">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>680</y>
      <width>66</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="activeIndecator_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>66</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>40</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Go to menu</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>610</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QComboBox" name="combinationTitles">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="value_1Titles">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="SuitTitles">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="value_2Titles">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="passButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>88</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(165, 29, 45);</string>
     </property>
     <property name="text">
      <string>pass</string>
     </property>
    </widget>
    <widget class="QPushButton" name="acceptButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>88</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(87, 227, 137);
color: rgb(97, 53, 131);</string>
     </property>
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>1640</x>
      <y>480</y>
      <width>120</width>
      <height>191</height>
     </rect>
    </property>
    <widget class="QPushButton" name="showButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>88</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:#4f0303}</string>
     </property>
     <property name="text">
      <string>show</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hideButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>88</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:#2CFA75}</string>
     </property>
     <property name="text">
      <string>active</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hideAllButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>88</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>hide</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1926</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
